carzamm/carzam
# Watch recognize crop

To test:
Ideally, you should setup a virtual environment before running this:
`python3 -m venv venv`
`source ./venv/bin/activate`

Install dependencies by doing
`pip3 install -r requirements.txt`

Then run by doing
`python3 carzam.py`

Then while Carzam is running, add a picture with a car or cars in it to ./data/testfiles

Carzam will automatically process the input file and output the cropped images to ./data/outfiles

I would like to get this merged to main. It's a significant step forward for us and we can begin integrating at least this part of it with the web application that Kevin is working on. Any input to my code is appreciated!
